karapace/rapu.py

```diff
@@ -393,11 +393,11 @@ async def _handle_request(
             )
             headers = {"Content-Type": "application/json"}
             resp = aiohttp.web.Response(body=body, status=status.value, headers=headers)
-        except ConnectionError as connection_error:
+        except (ConnectionError, aiohttp.ClientError) as connection_error:
```

Comment:
Do you really want to catch all the `aiohttp.ClientError` children? Or just `aiohttp.ClientConnectorError`?

Reply:
it could also probably be fine with the `ClientError` in general -> https://docs.aiohttp.org/en/stable/_modules/aiohttp/client_exceptions.html#ClientError

just wondering if all the subcases matches well with the 503

Reply:
I've gone with catching the base so as to clean up our issue tracker and we can always further investigate with the exception information added to the base, else, we'd need to branch for each possible connection error or client error, as we are the client in this case.